samples/sensor/max17048/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022, Alvaro Garcia <maxpowel@gmail.com>
```

Comment:
I don't think we need or want any of these KConfigs? None of the other drivers overwrite kconfigs.

Reply:
I agree